src/tools/perf/ucp_tests.cc

```diff
@@ -241,7 +218,7 @@ class ucp_perf_test_runner {
     }
 
     ucs_status_t UCS_F_ALWAYS_INLINE
-    recv(const recv_handle &hndl, void *buffer, unsigned length,
+    recv(ucp_worker_h worker, ucp_ep_h ep, void *buffer, unsigned length,
```

Comment:
What was the reason to remove `struct recv_handle`? Are there any benefits?

Reply:
reduce code size and simplify it, since this struct is not really needed

Reply:
for example, in common case, `ep` can be not initialized (current CM infrastructure doesn't allow this), the handle could protect from this in future.
If from your perspective that's ok to have not used or even not initialized parameter, i have no objections.

Reply:
i see that currently ep is always initialized.. and even if it wasn't, it would have still been passed to recv_handle constructor so recv_handle would also contain un-initialized value